examples/ffva/bsp_config/XK_VOICE_L71/platform/platform_start.c

```diff
@@ -45,15 +45,18 @@ static void flash_start(void)
 
 static void i2c_master_start(void)
 {
+#if !appconfI2C_DFU_ENABLED
     rtos_i2c_master_rpc_config(i2c_master_ctx, appconfI2C_MASTER_RPC_PORT, appconfI2C_MASTER_RPC_PRIORITY);
 
 #if ON_TILE(I2C_TILE_NO)
     rtos_i2c_master_start(i2c_master_ctx);
 #endif
+#endif
 }
 
 static void audio_codec_start(void)
 {
+#if !appconfI2C_DFU_ENABLED
```

Comment:
I have similar questions for this area as for line 48 above.

Reply:
See my comment above.